pom.xml

```diff
@@ -200,6 +200,11 @@
             <artifactId>azure-storage</artifactId>
             <version>${azure.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.synapse</groupId>
+            <artifactId>synapse-nhttp-transport</artifactId>
+            <version>2.1.7-wso2v19</version>
```

Comment:
please move the dependency version to the properties section of the pom

Reply:
Resolved on latest commit.